.eslintrc.js

```diff
@@ -16,8 +16,10 @@ module.exports = {
   },
   plugins: [
     '@typescript-eslint',
+    'unicorn',
   ],
   rules: {
+    'unicorn/prefer-node-protocol': 'error',
```

Comment:
Nice! Would be good to stick this rule last, they were meant to be alphabetically sorted (though I can see `semi` is in the wrong place...)